Sources/BraintreeVenmo/BTVenmoClient.swift

```diff
@@ -100,10 +99,9 @@ import BraintreeCore
             }
             
             do {
-                _ = try self.verifyAppSwitch(with: configuration, fallbackToWeb: request.fallbackToWeb)
+                _ = try self.verifyAppSwitch(with: configuration)
             } catch {
                 self.notifyFailure(with: error, completion: completion)
-                return
```

Comment:
I think we will want to add back this return, my guess is that this is why these tests are failing:
```
	BTVenmoClient_Tests.testTokenize_whenVenmoRequest_setsVaultAnalyticsTag()
	BTVenmoClient_Tests.testTokenizeVenmoAccount_whenVenmoConfigurationDisabled_callsBackWithError()
	BTVenmoClient_Tests.testTokenizeVenmoAccount_whenVenmoConfigurationDisabled_callsBackWithError()
	BTVenmoClient_Tests.testTokenizeVenmoAccount_whenVenmoConfigurationDisabled_callsBackWithError()
	BTVenmoClient_Tests.testTokenizeVenmoAccount_whenVenmoConfigurationMissing_callsBackWithError()
	BTVenmoClient_Tests.testTokenizeVenmoAccount_whenVenmoConfigurationMissing_callsBackWithError()
	BTVenmoClient_Tests.testTokenizeVenmoAccount_whenVenmoConfigurationMissing_callsBackWithError()
```

Reply:
Just added it back and tested locally. Seems like removing `return` was the cause of the failed tests